src/test/java/kitchenpos/application/OrderTableServiceTest.java

```diff
@@ -0,0 +1,209 @@
+package kitchenpos.application;
+
+
+import static kitchenpos.application.OrderTableFixture.삼번_테이블;
+import static kitchenpos.application.OrderTableFixture.일번_테이블;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import java.util.List;
+import kitchenpos.domain.Order;
+import kitchenpos.domain.OrderRepository;
+import kitchenpos.domain.OrderStatus;
+import kitchenpos.domain.OrderTable;
+import kitchenpos.domain.OrderTableRepository;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+import org.junit.jupiter.params.provider.EnumSource.Mode;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+
+@DisplayName("식탁 관리")
+class OrderTableServiceTest {
+
+    private final OrderTableRepository orderTableRepository = new InMemoryOrderTableRepository();
+    private final OrderRepository orderRepository = new InMemoryOrderRepository();
+    private OrderTableService orderTableService;
+
+    @BeforeEach
+    void setUp() {
+        orderTableService = new OrderTableService(orderTableRepository, orderRepository);
+    }
+
+    @DisplayName("식탁은 이름이 있어야 한다.")
+    @ParameterizedTest(name = "식탁 이름: [{arguments}]")
+    @NullAndEmptySource
+    void createException(String orderTableName) {
+        //given
+        OrderTable orderTable = 식탁_생성(orderTableName);
+
+        //when
+        ThrowingCallable actual = () -> orderTableService.create(orderTable);
+
+        //then
+        assertThatThrownBy(actual).isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @DisplayName("식탁을 생성한다. 식탁의 기본 상태는 0명의 손님이며 비어있다.")
+    @Test
+    void create() {
+        //given
+        OrderTable 신규_테이블 = 식탁_생성("3번");
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        OrderTable 신규_식탁 = 식탁_생성("3번");
```